src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -598,38 +598,56 @@ void destroyAllOntimeIntervalThreads(std::string tenant)
 
```

Comment:
Fixed in 2e65343 (it was easy).

After that, the set of failing tests is: 

```
----------- Failing tests ------------------
  o  117_convop_using_standard_ops/postContextEntitiesByEntityId.test
  o  117_convop_using_standard_ops/postContextEntityAttributes.test
  o  117_convop_using_standard_ops/postEntityByIdAttributeByName.test
  o  499_context_providers_basic/query_redirect_operation.test
  o  499_context_providers_basic/query_redirect_three_providers.test
  o  499_context_providers_basic/update_redirect_delete_operation.test
  o  499_context_providers_basic/update_redirect_operation.test
  o  714_propagate_service_path/no_propagation_of_service_path_when_not_present.test
  o  715_forward_xauth_token_to_cprs/propagate_xauth_token_for_update_and_query_standard_ops.test
  o  716_forward_registered_attributes/forward_attribute_query.test
  o  719_servicepath_for_registrations/forward_attribute_with_servicePath.test
  o  719_servicepath_for_registrations/forward_queries.test
```

All them corresponding to the CPs set.